common/oatbox/filesystem/FileSystemService.php

```diff
@@ -53,7 +53,28 @@ public function getFileSystem($id)
         return $this->filesystems[$id];
     }
     
-    public function addLocalFileSystem($id, $path)
+    /**
+     * Create a new local file system
+     * 
+     * @param string $id
+     * @param string $path
+     * @return Filesystem
+     */
+    public function createLocalFileSystem($id)
+    {
+        $path = $this->getOption(self::OPTION_FILE_PATH).\helpers_File::sanitizeInjectively($id);
+        $success = $this->addLocalFileSystem($id, $path);
```

Comment:
this method doesn't exist it should be ```registerLocalFileSystem```
Do you really need to get the return variable if you don't do anything with it ?